packages/api-specification/api/common.ts

```diff
@@ -0,0 +1,20 @@
+// contains common types
+export interface Bounds {
+    top: number;

```

Comment:
In Electron, `top` and `left` are `x` and `y`. If we are trying to stick to the Electron API as much as possible then maybe this should be changed?